ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -51,6 +63,17 @@ function MainPage(): ReactElement {
     const { isFeatureFlagEnabled, isLoadingFeatureFlags } = useFeatureFlags();
     const { hasReadAccess, hasReadWriteAccess, isLoadingPermissions } = usePermissions();
 
+    // Check for clusters under management
+    // if none, and user can admin Clusters, redirect to clusters section
+    // (only applicable in Cloud Services version)
```

Comment:
By "only applicable in Cloud Services version", does that mean that this situation is only ever possible in the cloud services version? Is there any way an on-prem install could return no clusters?

Related, is the ability to view clusters something that is controlled by SAC? Is it possible a user could have insufficient permissions to view clusters but still be able to do useful work, and this would cause them to get stuck on the clusters page?